app/js/scoreChart.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Function to update the chart. Due to how chart.js works, the only effective way to do
+ * this seems to be completely resetting the graph container every time.
+ */
+function updateChart() {
+    document.querySelector('.show-graph').innerHTML = '<canvas id="myChart"></canvas>';
+    drawChart();
+}
+
+/**
+ * script for graph using chart.js, links to canvas id=myChart in adminPage.html
+ */
+function drawChart() {
+    sendToChart().then((chartData)=>{
+        let ctx = document.getElementById('myChart');
+        // sets global font size and font family for the chart
+        Chart.defaults.global.defaultFontSize = 12;
+        Chart.defaults.global.defaultFontFamily = "'Asap', sans-serif";
+        var myChart = new Chart(ctx, {
+            type: 'bar',
+            // data and settings populating the chart
+            data: chartData,
+            options: {
+                title: {
+                    display: true,
+                    text: "Percentage score distribution across different aptitude tests",
+                    fontSize: 16,
+                },
+                scales: {
+                    yAxes: [{
+                        scaleLabel: {
+                            display: true,
+                            labelString: 'Number of tests taken',
+                            fontSize: 14
+                        },
+                        ticks: {
+                            beginAtZero: true
+                        }
```

Comment:
Can you set the y-axis so it shows whole numbers only?

Reply:
There's no inbuilt option for it, but I've done it using some code found on stackexchange: it uses a callback to only show the numbers on the y axis that are integers, and in this instance it appears to automatically truncate before the decimal point rather than showing 1.0, 2.0 etc.